nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/AbstractElasticsearchHttpProcessor.java

```diff
@@ -97,6 +116,27 @@
 
     private final AtomicReference<OkHttpClient> okHttpClientAtomicReference = new AtomicReference<>();
 
+    private static final List<PropertyDescriptor> propertyDescriptors;
+
+    static {
+        final List<PropertyDescriptor> properties = new ArrayList<>();
+        properties.add(ES_URL);
```

Comment:
Won't some of these be added twice, once by this abstract class and once by the individual processor? I like the idea of adding all common ones here, but I think they need to be removed from Fetch/PutESHttp

Reply:
Thanks @mattyb149 - I was not sure about what stand to make too.  I added them in the Abstract as a matter of convenience and to avoid duplication . Please let me know if you have any strong reservations, in which case, I'll go ahead and add it to all the inherited processors.

Reply:
I like them in the abstract class, but I believe they are also being added in some of the inherited processors IIRC. Also do you mind rebasing your PR against the latest master? There appear to be some merge conflicts. Please and thanks!

Reply:
Thanks @mattyb149 - contrib_checks and testcases passes after rebasing the latest from master. Appreciate if you could have a quick look again, please.